docs/en/latest/plugins/limit-count.md

```diff
@@ -79,7 +79,17 @@ curl -i http://127.0.0.1:9080/apisix/admin/routes/1 -H 'X-API-KEY: edd1c9f034335
 You can open dashboard with a browser: `http://127.0.0.1:9080/apisix/dashboard/`, to complete the above operation through the web interface, first add a route:
 ![Add a router.](../../../assets/images/plugin/limit-count-1.png)
 
-Then add limit-count plugin:
+## Then add limit-count plugin:
```

Comment:
Why make it a title?

Reply:
#### Then add limit-count plugin:

is this size good to go?
to make it lil bit more noticed